test/database.ts

```diff
@@ -3166,6 +3180,57 @@ describe('Database', () => {
       assert.strictEqual(options, fakeOptions);
     });
 
+    it('should optionally accept runner `option` isolationLevel', async () => {
+      const fakeOptions = {
+        isolationLevel:
+          protos.google.spanner.v1.TransactionOptions.IsolationLevel
+            .REPEATABLE_READ,
+      };
+
+      await database.runTransaction(fakeOptions, assert.ifError);
+
+      const options = fakeTransactionRunner.calledWith_[3];
+      assert.strictEqual(options, fakeOptions);
+    });
+
+    it('should optionally accept `option` isolationLevel when passed with Spanner options', () => {
```

Comment:
Add test cases for below 

1. Should override `isolationLevel` from runTransactionOptions, when passed in both `defaultTransactionOptions` and `runTransactionOptions`

2. Should use isolation level from `defaultTransactionOptions` when `runTransactionOptions` are passed but does not consists of isolation level and contains other properties. The resulting options should be a combination of `defaultTransactionOptions` and `runTransactionOptions`

Reply:
3. If `defaultTransactionOptions` has more properties except, isolation level, then other properties should be ignored.

Reply:
have added the tests 
1. 'Should override isolationLevel from runner `option`, when passed in both Spanner Options and runner `option`'
2. 'Should use isolationLevel from spanner `option`, when isolationLevel is passed at client `options` level and other options are passed at transaction `options` level'